examples/README_MODAL.md

```diff
@@ -0,0 +1,57 @@
+## Usage Examples
+
+This directory contains examples of running `marker` in different contexts.
+
+### Usage with Modal
+
+We have a [self-contained example](./modal_deployment.py) that shows how you can quickly use [Modal](https://modal.com) to deploy `marker` by provisioning a container with a GPU, and expose that with an API so you can submit PDFs for conversion into Markdown, HTML, or JSON.
+
+It's a limited example that you can extend into different use cases.
+
+#### Pre-requisites
+
+Make sure you have the `modal` client installed by [following their instructions here](https://modal.com/docs/guide#getting-started)
+
+#### Running the example
+
+Once `modal` is configured, you can deploy it to your workspace by running:
+
+> modal deploy marker_modal_deployment.py --env <YOUR_MODEL_ENV>
+
+Notes:
+- `marker` has a few models it uses. By default, the endpoint will check if these models are loaded and download them if not (first request will be slow). You can avoid this by running
+
+> modal run --env <YOUR_MODAL_ENV> modal_deployment.py::download_models
+
+Which will create a [`Modal Volume`](https://modal.com/docs/reference/modal.Volume) to store them for re-use.
+
+- Regardless, once the deploy is finished, you can submit a request. To do so, get the base URL for your endpoint:
```

Comment:
We often mention that the URL is present in the output when you `modal deploy`.

Also, you can add a simple client to a `local_entrypoint` to make trying the endpoint easier. I actually use this to add self-contained testing to Modal Apps.

In that code, you can introspect the URL of a Modal Function with `.get_web_url`. I sometimes print it in tutorials if I want to make it easier for folks to hit the endpoint with their own clients.